contracts/stake/src/multitest/delegate.rs

```diff
@@ -1,25 +1,29 @@
+use super::suite::{SuiteBuilder, SEVEN_DAYS};
 use cosmwasm_std::Uint128;
 use cw_controllers::Claim;
 
-use super::suite::{SuiteBuilder, SEVEN_DAYS};
+const AMOUNT: u128 = 100_000;
+const DENOM: &str = "VEST";
+const USER: &str = "user_addr_0000";
 
 #[test]
 fn delegate_and_unbond_tokens_still_vested() {
-    let user = "user";
+    let balances = vec![(USER, AMOUNT)];
     let mut suite = SuiteBuilder::new()
-        .with_initial_balances(vec![(user, 100_000)])
+        .with_native_balances(DENOM, balances)
+        .with_lp_share_denom(DENOM.to_string())
         .build();
 
     assert_eq!(
-        suite.query_balance_vesting_contract(user).unwrap(),
+        suite.query_balance_vesting_contract(USER).unwrap(),
         100_000u128
     );
 
     // delegate half of the tokens, ensure they are staked
-    suite.delegate(user, 50_000u128, None).unwrap();
-    assert_eq!(suite.query_staked(user, None).unwrap(), 50_000u128);
+    suite.delegate(USER, 50_000u128, None).unwrap();
+    assert_eq!(suite.query_staked(USER, None).unwrap(), 50_000u128);
```

Comment:
Don't hardcode the amounts for the whole file when you are using later some different amounts. It makes it more difficult to make sure the test is correct.

Reply:
I explicitly came back and made those as const values. Undo the one for amount, let me know if the rest needs to be undone as well.

Reply:
I'd say we shouldn't hardcode anything above the tests, because then you'll have some string in the error log and you can't see in the test case itself what exactly it is. And you have to scroll up.